fetch/main.js

```diff
@@ -0,0 +1,7 @@
+console.log(fetch('https://jsonplaceholder.typicode.com/users'));
+//   .then(response => response.json())
+//   .then(data => console.log(data));
+
+// fetch('https://pokeapi.co/api/v2/pokemon/150')
+//   .then(res => res.json())
+//   .then(data => console.log(data));
```

Comment:
This is the correct answer, fetch just returns a promise, not the data so you need the `.then`s to get to the data.